portal-web/test/functional/com/liferay/portalweb/macros/FormPortlet.macro

```diff
@@ -49,6 +49,8 @@ definition {
 			Click(locator1 = "FormPortlet#PORTLET_CONTENT");
 		}
 		else {
+			WaitForElementPresent(locator1 = "Form#PUBLISH_URL_FORM_PAGE");
+
```

Comment:
@carolmariaabb, are the modifications of this commit necessary due to the changes introduced by the last commits?

Reply:
@natocesarrego The first time I ran `ci:test:relevant` for this PR, the test had failed and because of that I made this correction. I will run without this commit and check if the failure is related.

Reply:
Ok. Please send me a new pull when you're ready. 😉